Packages/MIES/MIES_ExperimentDocumentation.ipf

```diff
@@ -491,7 +491,7 @@ static Function [WAVE colIndizes, variable rowIndex] ED_FindIndizesAndRedimensio
 
 	Make/FREE/D/N=(DimSize(incomingKey, COLS)) indizes = NaN
 
-	WAVE/T/Z desc
+	WAVE/T/ZZ desc
```

Comment:
cool!

Reply:
Jeep, getting into more and more IP9 only features.